backend/controller/sql/schema/001_init.sql

```diff
@@ -263,6 +263,27 @@ CREATE TABLE cron_jobs
 CREATE INDEX cron_jobs_executing_start_time_idx ON cron_jobs (start_time) WHERE state = 'executing';
 CREATE UNIQUE INDEX cron_jobs_key_idx ON cron_jobs (key);
 
+CREATE OR REPLACE FUNCTION create_cron_job(
+  k cron_job_key,
+  d_key deployment_key,
+  m_name TEXT,
+  v TEXT,
```

Comment:
Please give these parameters meaningful names. If they conflict with columns, just append an underscore or something.

Reply:
reverted to the pre-func version of this, but I'll keep the `_` prefix idea in mind for next time - thanks!